providers/implementations/encode_decode/encode_key2text.c

```diff
@@ -376,18 +376,17 @@ static int ec_param_explicit_curve_to_text(BIO *out, const EC_GROUP *group,
 static int ec_param_explicit_gen_to_text(BIO *out, const EC_GROUP *group,
                                          BN_CTX *ctx)
 {
+    int ret = 0;
```

Comment:
The initializers of ret and buflen are superfluous and will be reported by static analyzers. @paulidale were removing similar ones in another PR.

Reply:
I do it out of habit, for all the times where we have ended up with uninitialized values being used.
Changed - by not sure I agree with static analyzers doing this.

Reply:
That other PR says many of these are false positives.  The code is safer, and more robust if any internal code-paths change, with the initializer.  @slontis stay strong and keep it! :)

Reply:
There are arguments for both cases..
If someone changes the code then they probably should be checking the paths anyway.
I have plenty of other things to do in the meantime, so arguing about minor points that don't really matter is probably counter-productive :)